app/__init__.py

```diff
@@ -67,5 +68,17 @@
     # TODO - No need of this for REST
     app.logger.info('ProgDay startup')
 
+# defining the errors blueprint
+from app.errors import bp as errors_bp
+app.register_blueprint(errors_bp)
+
+# defining the auth blueprint
+from app.auth import bp as auth_bp
+app.register_blueprint(auth_bp, url_prefix='/auth')
+
 # import the main navigation control for the app
-from app import routes, models, errors
\ No newline at end of file
+from app import models
+
```

Comment:
All models will be imported in the endpoints no need to import them here. Only routes are mandatory